src/sass/modules/_m-form-elements.scss

```diff
@@ -142,13 +142,6 @@ button.form-button-disabled {
 }
 
 .form-expanding-group {
-  .usa-input-error {
```

Comment:
I must have added it a while back so that the bars lined up underneath a group. Not sure why it would be different now.